.github/ISSUE_TEMPLATE/BC_Break.md

```diff
@@ -4,7 +4,7 @@ about: Have you encountered an issue during an upgrade? 💣
 ---
 
 <!--
-Before reporting a BC break, please consult the upgrading document to make sure it's not an expected change: https://github.com/doctrine/dbal/blob/master/UPGRADE.md
+Before reporting a BC break, please consult the upgrading document to make sure it's not an expected change: https://github.com/doctrine/dbal/blob/4.0.x/UPGRADE.md
```

Comment:
Since this change is being made for the `2.12.x` branch, would it make sense to use something like "the upgrading document corresponding to the version you use" rather than a hard-coded `4.0.x`?